app/src/main/java/woowacourse/shopping/data/mapper/ProductMapper.kt

```
@@ -0,0 +1,25 @@
+        )
+    }
+
+fun Product.toEntity() =
+    ProductEntity(
+        id = this.id,
+        imageUrl = this.imageUrl,
+        name = this.name,
+        price = this.price,
+    )
+
+fun Long.toProductDomain() = ProductData.products.find { it.id == this } ?: throw IllegalArgumentException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)